### ISSUE TITLE
IDA-minsc compatibility with the IDAPython version from IDA 7.5.200619

### ISSUE THREAD

**Issue by arizvisa** [arizvisa/ida-minsc#68]
Minsc's documentation doesn't mention anything about compatibility with IDA 7.5. This is because it hasn't yet been tested on IDA 7.5 due to my personal hatred for IDA 7.x's licensing stupidities which makes it super-fucking expensive for me to test the extension on platforms that I don't use regularly.

Currently IDA 7.5 was related in 2020/5/20 (as per https://www.hex-rays.com/blog/ida-pro-7-5-released/), with SP1 in 2020/6/19 (as per https://www.hex-rays.com/blog/ida-pro-7-5-sp1-released/), and SP2 in 2020/7/28 (as per https://www.hex-rays.com/blog/ida-pro-7-5-sp2-released/).

**Comment by arizvisa** [comment]
Welp...the first error I get is:
```
Loading processor module /home/user/idapro-7.5/procs/pc.so for metapc...Initializing processor module metapc...OK
Exception in ida_idp.IDP_Hooks dispatcher function: SWIG director method error. Error detected when calling 'IDP_Hooks.ev_set_idp_options'
Traceback (most recent call last):
  File "/home/user/work/ida-minsc/base/_interface.py", line 426, in method
    return supermethod(*parameters, **keywords)
  File "/home/user/idapro-7.5/python/2/ida_idp.py", line 1841, in ev_set_idp_options
    return _ida_idp.IDP_Hooks_ev_set_idp_options(self, *args)
TypeError: in method 'IDP_Hooks_ev_set_idp_options', argument 4 of type 'void const *'
```

**Comment by arizvisa** [comment]
Documentation for this fucker is here: https://www.hex-rays.com/products/ida/support/idapython_docs/ida_idp.processor_t-class.html#ev_set_idp_options

We're getting called with the parameters `('MIPS_SIMPLIFY_GP', 3, 1, False)` which of course does not correspond to the parameters that we need to pass to `ev_set_idp_options` which has the prototype `const char *(idaapi set_options_t)(const char *keyword, int value_type, const void *value, bool idb_loaded)`. Second parameter is supposed to be a void* which SWIG doesn't seem to be converting properly.

So this means we have to fucking patch the way we call the parent method of this hook....because y'know...IDA.

**Comment by arizvisa** [comment]
Closing this issue as it has been fixed by PR #69.

This finally allows us to announce a new release for IDA 7.5 support.
